draft-ietf-quic-transport.md

```diff
@@ -806,7 +806,8 @@ A client MUST NOT reset the packet number it uses for 0-RTT packets.  The keys
 used to protect 0-RTT packets will not change as a result of responding to a
 Retry or Version Negotiation packet unless the client also regenerates the
 cryptographic handshake message.  Sending packets with the same packet number in
-that case would compromise the packet protection for all 0-RTT packets.
+that case could compromise the packet protection for all 0-RTT packets because
```

Comment:
Why the change from "would" to "could". Could seems a lot more likely.

Reply:
It's not guaranteed - if the exact same data is sent again, then there is no compromise.  And, from what I know of some implementations, that might even work.  But it's a long shot.  I'll tweak it to change emphasis.